system/autoware_state_monitor/src/autoware_state_monitor_node/autoware_state_monitor_node.cpp

```diff
@@ -269,19 +273,8 @@ void AutowareStateMonitorNode::onTimer()
 
   // Publish state message
   {
-    autoware_system_msgs::msg::AutowareState autoware_state_msg;
-    autoware_state_msg.state = toString(autoware_state);
-
-    // Add messages line by line
-    std::ostringstream oss;
-
-    oss << getStateMessage(autoware_state) << std::endl;
-
-    for (const auto & msg : state_machine_->getMessages()) {
-      oss << msg << std::endl;
-    }
-
-    autoware_state_msg.msg = oss.str();
```

Comment:
Could you remove `state.msg` from autoware_web_controller?

Reply:
in this PR?

Reply:
Or another PR is okay. I'll leave it to you!

Reply:
autoware_web_controller seems to have many other parts that need to be modified, so I will divide the PR